pig/src/main/java/org/jboss/pnc/bacon/pig/impl/repo/RepoManager.java

```diff
@@ -177,6 +177,9 @@ private RepositoryData packAllBuiltAndDependencies() {
 
     private RepositoryData buildGroup() {
         log.info("Generating maven repo for build group [{}]", pigConfiguration.getGroup());
+        if (builds.isEmpty()) {
```

Comment:
Is it possible for either of these to be empty under normal circumstances so that you would just want to skip them and not fail?

Reply:
Yes in weird circumstances that Paul Gallagher hit it might be empty.